test/integration/scan_test.go

```diff
@@ -1297,3 +1314,213 @@ func TestScanListWithFilters(t *testing.T) {
 	err, _ := executeCommand(t, args...)
 	assert.NilError(t, err, "")
 }
+
+func TestCreateScan_WithOnlyValidApikeyLabel_Success(t *testing.T) {
```

Comment:
```suggestion
func TestCreateScan_WithOnlyValidApikeyFlag_Success(t *testing.T) {
```
suggestion for all the tests 'flag' instead of 'Label'